hadoop-ozone/client/src/main/java/org/apache/hadoop/ozone/client/rpc/RpcClient.java

```diff
@@ -704,6 +704,21 @@ public void setBucketQuota(String volumeName, String bucketName,
 
   }
 
+  @Override
+  public void setReplicationConfig(
+      String volumeName, String bucketName, ReplicationConfig replicationConfig)
+      throws IOException {
+    verifyVolumeName(volumeName);
+    verifyBucketName(bucketName);
+    Preconditions.checkNotNull(replicationConfig);
+    OmBucketArgs.Builder builder = OmBucketArgs.newBuilder();
+    builder.setVolumeName(volumeName)
+        .setBucketName(bucketName)
+        .setDefaultReplicationCOnfig(
```

Comment:
There seems to be a typo in ".setDefaultReplicationCOnfig" where the "O" in config is in uppercase. Not sure if this is a pre-existing problem, as I don't see this method defined in this PR.